src/simularium/NetConnection.ts

```diff
@@ -108,6 +107,35 @@ export class NetConnection {
             return;
         }
 
+        if (event.data instanceof ArrayBuffer) {
+            const floatView = new Float32Array(event.data);
+            const binaryMsgType = floatView[0];
+
+            if (binaryMsgType === NetMessageEnum.ID_VIS_DATA_ARRIVE) {
+                const nameLength = floatView[1];
+                const byteView = new Uint8Array(event.data);
+                const fileBytes = byteView.subarray(8, 8 + nameLength);
```

Comment:
the above two lines might be more readable as 
`const OFFSET_TO_FILENAME = 8;`
`const fileNameBytes = new Uint8Array(event.data, OFFSET_TO_FILENAME, nameLength);`

Reply:
Will do a pass today on both PRs to replace the magic numbers